app/views/layouts/application.html.slim

```diff
@@ -26,6 +26,9 @@ html lang="#{I18n.locale}"
       <script src='https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js'></script>
     <![endif]-->
 
+    - if Rails.env.production? && Figaro.env.google_analytics_key.present?
```

Comment:
Do we want this keyed on `Rails.env.production?` -- i.e. is there ever a time we would have a different Rails.env that needed GA? Seems like `Figaro.env.google_analytics_key.present?` should be enough.